tests/integration/tasksManager/tasksManager.spec.ts

```diff
@@ -24,11 +23,9 @@ describe('resourceName', function () {
       const response = await requestSender.getResource();
 
       expect(response.status).toBe(httpStatusCodes.OK);
-      expect(response).toSatisfyApiSpec();
-
-      const resource = response.body as IAnotherResourceModel;
-      expect(resource.kind).toBe('avi');
-      expect(resource.isAlive).toBe(false);
+      expect(response).toSatisfyApiSpec(); // TODO: use when tests are done
+      // eslint-disable-next-line @typescript-eslint/no-magic-numbers
+      expect(1 + 1).toBe(2);
```

Comment:
?

Reply:
just for the exmaple, didnt want to remove those tests